src/components/molecules/Personalisation/Personalisation.tsx

```diff
@@ -0,0 +1,49 @@
+import concatClassNames from '@/src/utils/concatClassNames';
+import styles from './Personalisation.module.scss';
+import content from '@/src/content/content';
+
+const personalisationContent = content.components.personalisationComponent;
+
+export type PersonalisationType = {
+  isOpen?: boolean;
+};
+
+export function Personalisation({ isOpen }: PersonalisationType) {
+  return (
+    <>
+      <h2
+        className='nhsuk-heading-m nhsuk-u-margin-top-4'
+        data-testid='personalisation-header'
+      >
+        {personalisationContent.header}
+      </h2>
+      <details
```

Comment:
just using the Details component from nhsuk-react-components will simplify a lot of this

Reply:
➕